Formula/asdf.rb

```diff
@@ -13,8 +13,12 @@ def install
     bash_completion.install "completions/asdf.bash"
     fish_completion.install "completions/asdf.fish"
     zsh_completion.install "completions/_asdf"
-    libexec.install Dir["*"]
-    bin.write_exec_script (libexec/"bin/asdf")
+    bin.write_exec_script (opt_libexec/"bin/asdf")
+    prefix.install_metafiles
```

Comment:
```suggestion
```
`brew` should be doing this automatically already.

Reply:
@carlocab thanks!